src/clr-addons/searchfield/search-field.ts

```diff
@@ -52,11 +53,18 @@ export class ClrSearchField implements OnInit, OnDestroy, AfterViewInit {
     const inputWrapper = this.inputEl.nativeElement.parentNode;
 
     if (!this.deleteSymbol) {
+      this.deleteButton = this.renderer.createElement('button');
+      this.renderer.setAttribute(this.deleteButton, 'type', 'button');
+      this.renderer.addClass(this.deleteButton, 'btn');
+      this.renderer.addClass(this.deleteButton, 'btn-link');
+      this.renderer.addClass(this.deleteButton, 'delete-button');
+
       this.deleteSymbol = this.renderer.createElement('clr-icon');
       this.renderer.setAttribute(this.deleteSymbol, 'shape', 'times');
       this.renderer.addClass(this.deleteSymbol, 'remove-symbol');
       this.deleteSymbol.addEventListener('click', () => this.clearSearchInput());
```

Comment:
event listener needs to be on deleteButton now, otherwise the clear won't work anymore